jsk_pr2_robot/jsk_pr2_startup/pr2_bringup.launch

```diff
@@ -18,7 +18,8 @@
 
   <!-- pr2_etherCAT -->
   <node name="realtime_loop" machine="c1" launch-prefix="pr2-grant" pkg="pr2_ethercat" type="pr2_ethercat" args="-i ecat0 -x robot_description" output="screen" />
-  <param name="pr2_etherCAT/realtime_socket_timeout" value="5000" />
+  <param name="pr2_ethercat/realtime_socket_timeout" value="20000" />
```

Comment:
Please add a comment to describe this change.

Reply:
thanks, i add comment